src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateProbationBookingService.kt

```diff
@@ -45,7 +43,6 @@ class CreateProbationBookingService(
     }
 
     val probationTeam = probationTeamRepository.findByCode(request.probationTeamCode!!)
-      ?.also { require(it.enabled) { "Probation team with code ${it.code} is not enabled" } }
```

Comment:
Non-enabled are allowed in the new world e.g. prison users that do not know the probation team.

Technically this could check the user type as well as enabled I suppose, meh!